src/frontend/specification/EnsembleParameterTable.jl

```diff
@@ -2,50 +2,73 @@
 ### Types
 ############################################################################################
 
-mutable struct EnsembleParameterTable{C} <: AbstractParameterTable
-    tables::C
+struct EnsembleParameterTable{K} <: AbstractParameterTable
+    tables::Dict{K, ParameterTable}
+    params::Vector{Symbol}
```

Comment:
I guess we could just decide that the keys should be Symbols?
```suggestion
struct EnsembleParameterTable <: AbstractParameterTable
    tables::Dict{Symbol, ParameterTable}
    params::Vector{Symbol}
```